### PR TITLE
Minor Tcomms Update: Decrufts Syndicate Comms, Removes Station Frequency Names, and Expands Usable Frequencies

### PR DESCRIPTION
**Pull request by firebudgy**
## About The Pull Request

Telecommunications was obtuse and adding a faction's custom frequency required codediving or knowledge ahead of time when working with machines. Additionally, departmental comms (including Command) have been dead for years at this point and won't make a return. This correctly mataches frequencies to their factions and reflavors Command to an Emergency Frequency (a reference to https://en.wikipedia.org/wiki/Aircraft_emergency_frequency). Pirates are now only listed as "Unidentified" rather than Pirates.

Updated channels listed here.
image

Another major change is the removal of special treatment for syndicate comms. No longer can they listen in on every frequency, nor are they prevented from having their frequency tuned into. Theirs is the exact same as everyone else's, now.

This also expands the list of usable frequencies by a decent margin in order to make it significantly less likely that you can accidentally tune into someone else's comms, as well as specifically make it so certain powergaming strategies are no longer feasible. Wideband has been moved up to compensate.

## Why It's Good For The Game

Removes a large amount of residual references to station content and makes Syndicate communications not blatantly able to monitor every single channel. Makes tcomms easier to work with when adding channels, and organization within servers easier. Stops certain powergaming strategies from being feasible while reducing the constant frequency overlaps from only having 24 bands (dating back to the existence of stations).

## Changelog

:cl: Cloudbreak
del: AI Private frequency
balance: Syndicate communications are no longer able to monitor all other factions.
balance: Syndicate communications are no longer unable to be monitored.
balance: Nanotrasen has modernized their handhelds. Possible radio frequencies adjusted from 144.1-148.9 to 144.1-168.9. 
add: Reflavored set channels to properly ID their faction.
fix: Vox sprite compatibility in a single sprite.
/:cl:

### CODE REVIEW

**Comment by Orchidthederg** [comment]
FINALLY. THANK YOU. THANK YOU SO MUCH. AUGHGHHH THANKYOUTHANKYOUTHANKYOU I'M CRYING

File: code/modules/clothing/under/syndicate.dm [hunk 1]
```diff
@@ -267,7 +267,7 @@
 	name = "inteq corpsman skirtleneck"
 	desc = "A sterile white turtleneck with a free flowing black skirt, it is emblazoned with the lettering 'IRMG' on the shoulder. For the IRMG's support division Corpsmen."
 	icon_state = "inteqmed_skirt"
-	supports_variations = KEPORI_VARIATION | DIGITIGRADE_VARIATION_NO_NEW_ICON
+	supports_variations = KEPORI_VARIATION | VOX_VARIATION | DIGITIGRADE_VARIATION_NO_NEW_ICON
```
Comment (Anticept) [inline code/modules/clothing/under/syndicate.dm:259]: I think this got mixed in.

Comment (firebudgy) [reply code/modules/clothing/under/syndicate.dm:259]: Accidental addition, but I suppose it's a fix part of the PR now. Resolved without a comment, my bad.

**Comment by meemofcourse** [comment]
you should change the minutemen frequency to CLIP (or just C-MM), because i believe their full name is CLIP Minutemen

**Comment by firebudgy** [comment]
> you should change the minutemen frequency to CLIP (or just C-MM), because i believe their full name is CLIP Minutemen

Done, changed to Minutemen. In line with their channel in-game which is labeled as "Minutemen".

**Review by MarkSuckerberg — APPROVED** [review]

**Comment by thgvr** [comment]
Mark approved it and I'm not brave enough to gaze into tcomms code

**Pull request merged**